ledger/simulation/trace.go

```diff
@@ -51,9 +50,6 @@ type TxnGroupResult struct {
 
 	// AppBudgetConsumed is the total opcode cost used for this group
 	AppBudgetConsumed uint64
-
-	// FeeCredit is the fees left over after covering fees for this group
-	FeeCredit uint64
```

Comment:
I discovered this field was unused

Reply:
Yes. I believe it was introduced in #5221. I discovered this field is set by the tracer, but not used in any other way. @algochoi do you have any thoughts about this?

Reply:
Ok thanks. This likely will be useful at some point, but until then I'd like to remove it just to keep things clean